src/overlays/actors/ovl_Obj_Bombiwa/z_obj_bombiwa.c

```diff
@@ -22,18 +30,128 @@ const ActorInit Obj_Bombiwa_InitVars =
     (ActorFunc)ObjBombiwa_Update,
     (ActorFunc)ObjBombiwa_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Bombiwa/func_80B915F0.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Bombiwa/ObjBombiwa_Init.s")
+static ColliderCylinderInit colliderInit =
+{
+    0x0C, 0x00, 0x0D, 0x39, 
+    0x20, 0x01, 0x00, 0x00, 
+    0x00, 0x00, 0x00, 0x00, 
+    0x00000000, 0x00, 0x00, 
+    0x00, 0x00, 0x4FC1FFFE, 
+    0x00, 0x00, 0x00, 0x00, 
+    0x00, 0x01, 0x01, 0x00,
+    0x0037, 0x0046, 0x0000, 
+    0x0000, 0x0000, 0x0000
```

Comment:
I believe we want to start adding trailing commas to multi-line arrays, but confirm with roman